trunk/3rdparty/st-srs/stk.c

```diff
@@ -72,14 +74,34 @@ _st_stack_t *_st_stack_new(int stack_size)
             _st_num_free_stacks--;
             ts->links.next = NULL;
             ts->links.prev = NULL;
-            return ts;
+            returnt ts;
```

Comment:
is it a mistake? `returnt` -> `return`.

Reply:
It's a mistake.
To reproduce, I need to add `-DMD_CACHE_STACK` to `auto/depends.sh`, run `make clean_st`, then `./configure && make`. 

https://github.com/ossrs/srs/blob/ea7e2c28491ae61cc1b7aaa207ce38a74fa36487/trunk/auto/depends.sh#L283-L297

`st-srs` never spend too much time to build, maybe need to consider whether it's a good idea to build st-srs in this inconvenient way.

Reply:
Fixed.

BTW: You can use configure to enable the macro:

```
./configure -h |grep flags
#  --extra-flags=<EFLAGS>    Set EFLAGS as CFLAGS and CXXFLAGS. Also passed to ST as EXTRA_CFLAGS.
```